packages/editor/src/components/toolbar/tools/PlayModeTool.tsx

```diff
@@ -81,6 +82,8 @@ const PlayModeTool = () => {
       // getMutableState(EngineState).isEditing.set(false)
       // run all visual script logic
       visualScriptQuery().forEach((entity) => dispatchAction(VisualScriptActions.execute({ entity })))
+      transformGizmoControlledQuery().forEach((entity) => removeComponent(entity, TransformGizmoControlledComponent))
```

Comment:
Sadly I think you still have to make it that they can not appear when play mode is active. It does remove the transformGizmo but if I click on the heirachy it still creates a new transform gizmo.

Reply:
Looking really really good though. It's UX is flawless. I just wanna cover this one edge case and lets merge.